etc/behat/suites/users.yml

```diff
@@ -14,5 +14,12 @@ default:
                 - Sylius\Bundle\WebBundle\Behat\WebContext
                 - Sylius\Bundle\MoneyBundle\Behat\MoneyContext
                 - Sylius\Bundle\ChannelBundle\Behat\ChannelContext
+                - Sylius\Behat\Context\Setup\UserContext:
```

Comment:
Yes, because of this step:  https://github.com/Sylius/Sylius/pull/4009/files#diff-dfd9a9cd210e57337cdf7004fb01508fR152